data/data.go

```diff
@@ -31,7 +32,7 @@ func GetCluster(
 	}
 	err = AddUpdateData(&cluster, v, secretGetterCreator)
 	if err != nil {
-		return models.Cluster{}, err
+		log.Println("unable to decorate cluster data with available updates data")
```

Comment:
When we're unable to add definitive "available component update" data we'll log the event, but we don't want to short-circuit the entire version data-gathering process.